include/joypad.h

```diff
@@ -0,0 +1,427 @@
+/**
+ * @file joypad.h
+ * @author Christopher Bonhage (me@christopherbonhage.com)
+ * @brief Joypad Subsystem
+ * @ingroup joypad
+ */
+
+#ifndef __JOYPAD_H
```

Comment:
We're standardizing on using this convention:
```suggestion
#ifndef LIBDRAGON_JOYPAD_H
```

Reply:
Same for all other include files

Reply:
Good catch: this was an artifact from copying over the files from [my JoypadTest-N64 repo](https://github.com/meeq/JoypadTest-N64). Will fix.

Reply:
Fixed in 6357b4aa59874d828cd04a62355aefb90294c3f7